client/components/editor/structure/TesList.vue

```diff
@@ -83,22 +77,18 @@ export default {
     },
     onHiddenElementDrawer() {
```

Comment:
Can we rename this to `onCloseElementDrawer` to maintain parity with `showElementDrawer`? :bowtie:
Also, it _requires_ changing event from `hidden` to `close`:
```vue
@close="onCloseElementDrawer"
```